workflows/proteomics/clinicalmp/clinicalmp-quantitation/iwc-clinicalmp-quantitation.ga

```diff
@@ -0,0 +1,404 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "Clinical Metaproteomics ",
+    "comments": [],
+    "creator": [
+        {
+            "class": "Organization",
+            "name": "GalaxyP"
+        }
+    ],
+    "format-version": "0.1",
+    "license": "CC-BY-4.0",
+    "name": "iwc-clinicalmp-quantitation",
```

Comment:
```suggestion
    "name": "Clinical Metaproteomics Quantitation",
```
This is the name that users see in the Galaxy interface. The owner of the workflow will be shown as IWC so I don't think we need to have it in the name as well.